src/App.js

```diff
@@ -23,41 +24,42 @@ function App() {
     <div className="App">
       <Routes>
         <Route element={<SidebarLayoutPage />}>
-          <Route path="/" element={<Homepage />}></Route>
+          <Route path="/" element={<Homepage />} />
           <Route
             path="/likes"
             element={
               <RequireAuth>
                 <Likepage />
               </RequireAuth>
             }
-          ></Route>
+          />
           <Route
             path="/playlist"
             element={
               <RequireAuth>
                 <Playlistpage />
```

Comment:
instead of wrapping require auth on every element make a root level route and add the require auth element and then add all these component (use Outlet) inshort